frontend/src/container/CreateAlertChannels/index.tsx

```diff
@@ -293,6 +309,43 @@ function CreateAlertChannels({
 		setSavingState(false);
 	}, [prepareOpsgenieRequest, t, notifications]);
 
+	const prepareEmailRequest = useCallback(
+		() => ({
+			name: selectedConfig?.name || '',
```

Comment:
We can skip the safety check and fallback for mandatory fields

Reply:
It's the safety check for selectedConfig.